core/src/sql/value/value.rs

```diff
@@ -2573,6 +2661,19 @@ impl Value {
 				Value::Geometry(w) => v.contains(w),
 				_ => false,
 			},
+			Value::Range(r) => {
```

Comment:
@emmanuel-keller how can we incorporate this addition into the query planner (?)

Reply:
To be implemented during beta. CC @emmanuel-keller